invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/search.html

```diff
@@ -40,8 +40,13 @@
     },
     {
       "aggName": "subjects",
-      "field": "subjects",
-      "title": "{{_("Subjects")}}"
+      "field": "subjects.scheme",
+      "title": "{{_("Subjects")}}",
+      "childAgg": {
+        "aggName": "inner",
+        "field": "subjects.subject",
+        "title": "{{_("Resource types")}}"
```

Comment:
This title is a typo right?
```suggestion
        "title": "{{_("Subjects")}}"
```

Reply:
Yep thanks! (I don't know where it shows up on the UI :shrug: )